mmdet3d/datasets/pipelines/formating.py

```diff
@@ -264,3 +265,112 @@ def __repr__(self):
         repr_str += f'(class_names={self.class_names}, '
         repr_str += f'with_gt={self.with_gt}, with_label={self.with_label})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class GetBEVDetInputs():
```

Comment:
Maybe this transformation can be used in other methods. Could we rename it to a more general and clear name? For example,  `PackLSSInfos`.